ext/openssl/openssl.stub.php

```diff
@@ -95,105 +92,100 @@ function openssl_pkey_free(OpenSSLAsymmetricKey $key): void {}
+function openssl_private_encrypt(string $data, &$encrypted_data, $private_key, int $padding = OPENSSL_PKCS1_PADDING): bool {}
 
 /**
- * @param string $crypted
- * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $key
+ * @param string $encrypted_data
+ * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $private_key
  */
-function openssl_private_decrypt(string $data, &$crypted, $key, int $padding = OPENSSL_PKCS1_PADDING): bool {}
+function openssl_private_decrypt(string $data, &$encrypted_data, $private_key, int $padding = OPENSSL_PKCS1_PADDING): bool {}
 
 /**
- * @param string $crypted
- * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $key
+ * @param string $encrypted_data
+ * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $public_key
  */
-function openssl_public_encrypt(string $data, &$crypted, $key, int $padding = OPENSSL_PKCS1_PADDING): bool {}
+function openssl_public_encrypt(string $data, &$encrypted_data, $public_key, int $padding = OPENSSL_PKCS1_PADDING): bool {}
 
 /**
- * @param string $crypted
- * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $key
+ * @param string $encrypted_data
+ * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $public_key
  */
-function openssl_public_decrypt(string $data, &$crypted, $key, int $padding = OPENSSL_PKCS1_PADDING): bool {}
+function openssl_public_decrypt(string $data, &$encrypted_data, $public_key, int $padding = OPENSSL_PKCS1_PADDING): bool {}
 
 function openssl_error_string(): string|false {}
 
 /**
  * @param string $signature
- * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $key
- * @param int|string $method
+ * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $private_key
  */
-function openssl_sign(string $data, &$signature, $key, $method = OPENSSL_ALGO_SHA1): bool {}
+function openssl_sign(string $data, &$signature, $private_key, string|int $algorithm = OPENSSL_ALGO_SHA1): bool {}
 
-/**
- * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $key
- * @param int|string $method
- */
-function openssl_verify(string $data, string $signature, $key, $method = OPENSSL_ALGO_SHA1): int|false {}
+/** @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $public_key */
+function openssl_verify(string $data, string $signature, $public_key, string|int $algorithm = OPENSSL_ALGO_SHA1): int|false {}
 
 /**
- * @param string $sealdata
- * @param array $ekeys
- * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $pubkeys
- * @param string $iv
+ * @param string $sealed_data
+ * @param array $encrypted_keys
+ * @param string $initialization_vector
  */
-function openssl_seal(string $data, &$sealdata, &$ekeys, array $pubkeys, string $method, &$iv = UNKNOWN): int|false {}
+function openssl_seal(string $data, &$sealed_data, &$encrypted_keys, array $public_key, string $cipher_algorithm, &$initialization_vector = null): int|false {}
```

Comment:
```suggestion
function openssl_seal(string $data, &$sealed_data, &$encrypted_keys, array $public_key, string $cipher_algorithm, &$iv = null): int|false {}
```
IV is typically not spelled out.

Reply:
I don't feel particularly strongly about this one though. Maybe @bukka has a preference.

Reply:
Yes this should be changed to `iv`

Reply:
I'll do it soon